en/news/_posts/2019-04-01-ruby-2-6-3-released.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: news_post
+title: "Ruby 2.6.3 Released"
+author: "naruse"
+translator:
+date: 2019-04-01 00:00:00 +0000
+lang: en
+---
+
+Ruby 2.6.3 has been released.
+
+This release update Unicode version to 12.1, which add to New Japanese Era "〓〓".
```

Comment:
which adds support for

Reply:
```diff
- This release update
+ This release updates
```

Reply:
It also should have a `the` before 'Unicode version': 

```suggestion
This release updates the Unicode version to 12.1, which adds support for New Japanese Era "〓〓".
```